tool/state-processor/src/system/mod.rs

```diff
@@ -125,54 +125,25 @@ impl Processor {
 				// If the evm account is a contract contract with sufficients, then we should
 				// increase the sufficients by one.
 				if self.solo_state.contains_key(&full_key(
-					b"EVM",
+					b"Evm",
 					b"AccountCodes",
 					&blake2_128_concat_to_string(k),
 				)) && a.sufficients == 0
 				{
 					a.sufficients += 1;
 				}
 
-				if v.kton != 0 || v.kton_reserved != 0 {
-					let aa = AssetAccount {
-						balance: v.kton,
-						is_frozen: false,
-						reason: ExistenceReason::Sufficient,
-						extra: (),
-					};
-
-					a.sufficients += 1;
-					kton_details.accounts += 1;
-					kton_details.sufficients += 1;
-					// Note: this is double map structure in the pallet-assets.
+				if v.kton != 0 {
 					self.shell_state.insert_value(
 						b"Assets",
 						b"Account",
 						&format!(
 							"{}{}",
-							array_bytes::bytes2hex("", blake2_128_concat(&KTON_ID.encode())),
-							array_bytes::bytes2hex("", blake2_128_concat(&k.encode())),
+							blake2_128_concat_to_string(KTON_ID.encode()),
+							blake2_128_concat_to_string(k.encode()),
 						),
-						&aa,
-					);
-
-					// https://github.dev/darwinia-network/darwinia-common/blob/6a9392cfb9fe2c99b1c2b47d0c36125d61991bb7/frame/dvm/evm/precompiles/kton/src/lib.rs#L72
-					let mut approves = Map::<primitive_types::U256>::default();
```

Comment:
Why remove this?  Maybe no data in the Crab and Darwinia network now. But better have than not.

Reply:
The storage type double map. You can't use take map to process it.

And in the latest version, it is n map.

I prefer to reset the approvals.